lib/cloud_controller/packager/local_bits_packer.rb

```diff
@@ -4,20 +4,33 @@
 
 module CloudController
   module Packager
+    class ConflictError < StandardError
+    end
+
     class LocalBitsPacker
       include Packager::SharedBitsPacker
 
       def send_package_to_blobstore(blobstore_key, uploaded_package_zip, cached_files_fingerprints)
         tmp_dir = VCAP::CloudController::Config.config.get(:directories, :tmpdir)
-        Dir.mktmpdir('local_bits_packer', tmp_dir) do |root_path|
-          complete_package_path = match_resources_and_validate_package(root_path, uploaded_package_zip, cached_files_fingerprints)
+        local_bits_packer_path = File.join(tmp_dir, "local_bits_packer-#{blobstore_key}")
+
+        begin
+          Dir.mkdir(local_bits_packer_path)
+        rescue StandardError => e
+          raise ConflictError.new("Found a leftover directory that might be from the previous worker’s unfinished job: #{e.message}")
```

Comment:
**thought** We don't know for sure this error was generated due to a leftover directory since we're rescuing a generic StandardError right? Would it be more precise to explicitly check if the directory exists before throwing a Conflict error?

Reply:
I appreciate your suggestion and will make the necessary updates.

Reply:
Implemented in 042970dc6f70b8c4915cc25bdd91ea5607abfcd2